packages/web-components/src/components/va-radio-option/va-radio-option.tsx

```diff
@@ -124,8 +124,10 @@ export class VaRadioOption {
           id={id}
         >
           <label htmlFor={id} id="option-label">
-            {label}
-            {description && <span class="description" aria-describedby="option-label">{description}</span>}
+            <div>
+              {label}
+              {description && <span class="description" aria-describedby="option-label">{description}</span>}
+            </div>
```

Comment:
Wrapping the label and description in a `div` otherwise flex box would try to keep all 3 on the same line